src/runtime_src/core/pcie/driver/linux/xocl/subdev/mb_scheduler.c

```diff
@@ -4871,7 +4871,7 @@ kds_custat_show(struct device *dev, struct device_attribute *attr, char *buf)
 		for (;idx < (exec->num_cus + exec->num_sk_cus); ++idx) {
 			sz += sprintf(buf+sz, "CU[@0x0] : %d status : %d name : %s\n",
 				      ert_cu_usage(xert, idx),
-				      ert_cu_status(xert, idx) ? AP_START : AP_IDLE,
+				      ert_cu_status(xert, idx),
```

Comment:
This seems break the AP_IDLE reporting? It used to be '0' representing AP_IDLE (0x4). But now, how are we going to show 0x4 status?

Reply:
It is working fine.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Compute Unit Status
CU[  0]: verify:verify_1                 @0x1400000         (IDLE)
SCU[ 0]: hello_world                     @N/A               (--)
SCU[ 1]: hello_world                     @N/A               (--)
SCU[ 2]: hello_world                     @N/A               (CRASHED)
SCU[ 3]: hello_world                     @N/A               (--)
SCU[ 4]: hello_world                     @N/A               (--)
SCU[ 5]: hello_world                     @N/A               (--)
SCU[ 6]: hello_world                     @N/A               (--)
SCU[ 7]: hello_world                     @N/A               (--)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~